index.js

```diff
@@ -0,0 +1,66 @@
+const createUser = require('./api_requests/createUser')
+const viewingPeeps = require('./api_requests/viewingPeeps')
+
+//let contentsDiv = document.querySelector("#contents")
+
+let peepContainer = () => { 
+  let container = document.createElement("div") 
+  container.id = "peepContainer"
+  //container.classList.add("container")
+  document.body.appendChild(container)
+}
+
+peepContainer()
+
+viewingPeeps((data) => {
+  data.forEach(x => {
+    console.log(x)
+    let div = document.createElement("div")
+    div.innerText = x.body
+    div.id = `peep${x.id}`
+    div.class = "peep"
+    document.getElementById('peepContainer').appendChild(div)
+    //document.getElementById(`#peep${x.id}}`).innerText = x.body
+  })
+})
+
+let newPeepContainer = () =>{
+  let newContainer = document.createElement("div")
+  newContainer.id = "newPeepContainer"
+  newContainer.classList.add("container")
+  document.body.prepend(newContainer)
+}
+
+newPeepContainer()
+
+
+let createPeep = document.getElementById('createpeep')
+let peepBoxState = false 
+
+
+createPeep.addEventListener("click", () =>{
```

Comment:
I like the functionality behind this. It works quite smoothly - when you click on 'new peep' a 'whats on your mind?' form pops up. Clicking on 'send' however does not add the peep to the list of peeps. I realise this is pull request is a partial solution though. Good job!